### PR TITLE
Read me Updates

### PR DESCRIPTION
**Pull request by kushaggarwal**
I have updated the readme of you site with the basic information about your society with image and quick links to your social account to give a broad overview of you plus I have maintained a field consisting of some of your previous bootcamps and the technology you work with
Please check for the same 
ThankYou

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,37 @@
 # cosc-website
 
-/index
-/about
-/coc
-/privacy
-/djangocamp
+![alt text](./img/cosc.png)
```
Comment (r0mflip) [inline README.md]: Use a better alt-text say the below one, and use 
```suggestion
![COSC banner](img/banner.svg)
```

**Comment by r0mflip** [comment]
We have updated our PR template, please have a look at it. https://github.com/cbitosc/cbitosc.github.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: README.md (hunk 1 above)
Comment (kushaggarwal) [reply README.md]: Ok sure will update the following changes

**Comment by kushaggarwal** [comment]
I have updated the changes please have a look 
thank you

**Comment by r0mflip** [comment]
Done. Thanks.

**Comment by kushaggarwal** [comment]
Your Welcome 
Happy to help

**Pull request merged**